project-demo.md

```diff
@@ -0,0 +1,10 @@
+# Demo
+
+Given 4 CLI agents:
```

Comment:
Nice! Very interessting approach!
The name, that is a credential itself right?

Reply:
Yes. I should probably add an explanation of each credential.

NAME - A credential signed by the GOVERNMENT, letting everyone know what your official first name(s) + family name(s) are.